XenditExample/XenditExample/Controllers/CreateAuthentification.swift

```diff
@@ -14,6 +14,10 @@ class CreateAuthentification: UIViewController {
     
     @IBOutlet weak var tokenIDTextField: UITextField!
     @IBOutlet weak var amountTextField: UITextField!
+    @IBOutlet weak var cardHolderFirstNameTextField: UITextField!
+    @IBOutlet weak var cardHolderLastNameTextField: UITextField!
+    @IBOutlet weak var cardHolderEmailTextField: UITextField!
+    @IBOutlet weak var cardHolderPhoneNumberTextField: UITextField!
```

Comment:
If I am not mistaken, these are just outlets, you need to also update the storyboard and add actual fields for input
`/XenditExample/Base.lproj/Main.storyboard`
You can see from the storyboard xml, other fields are there, for example `tokenIDTextField`